### ISSUE TITLE
Where is South Sudan?

### ISSUE THREAD

**Issue by strawberry-code** [amcharts/amcharts4#2414]

**Comment by martynasma** [comment]
Last time I checked, South Sudan was in Central Africa ;)

**Comment by strawberry-code** [comment]
@martynasma 😂

However, you can use this:

```
{"type":"FeatureCollection", "features": [
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[29.80938,4.56455],[29.95484,4.65076],[30.07844,4.6475],[30.2053,4.7093],[30.29962,4.68328],[30.3972,4.60847],[30.5143,4.67027],[30.53707,4.80038],[30.70295,4.84917],[30.80379,4.93048],[30.86233,4.93048],[30.91112,5.03457],[30.89161,5.08336],[30.90462,5.15166],[30.93714,5.15166],[30.96317,5.20696],[30.93064,5.25575],[30.92413,5.38586],[30.99569,5.38586],[31.01846,5.57451],[30.65416,5.74365],[30.30394,6.08125],[30.52406,6.09168],[30.61188,6.25106],[31.02171,5.90953],[31.69732,5.87275],[32.11442,5.85005],[32.10224,5.36788],[32.09834,5.21346],[32.17315,4.95325],[32.12436,4.59871],[32.15039,4.55318],[32.00727,4.5174],[32.06582,4.47836],[32.07232,4.27019],[32.04955,4.23767],[31.99751,4.24417],[31.9845,4.10431],[31.86415,4.03926],[31.76332,4.0295],[31.72429,4.00022],[31.79958,3.82704],[31.80075,3.80105],[31.75565,3.76848],[31.7109,3.71717],[31.68602,3.70922],[31.66034,3.72068],[31.52745,3.65549],[31.40192,3.73103],[31.29094,3.79305],[31.17485,3.79563],[31.10907,3.75452],[31.09892,3.73679],[31.07415,3.74559],[31.05941,3.72132],[31.03338,3.70103],[30.98527,3.69954],[30.95451,3.67102],[30.94674,3.63616],[30.90736,3.59662],[30.88735,3.55953],[30.86915,3.50019],[30.84928,3.4902],[30.86273,3.55222],[30.84975,3.58453],[30.81414,3.58819],[30.79881,3.60352],[30.7981,3.65706],[30.77688,3.68111],[30.75542,3.66059],[30.75612,3.6363],[30.7316,3.62545],[30.68844,3.64385],[30.63938,3.61979],[30.62759,3.60682],[30.58844,3.61083],[30.5592,3.62168],[30.57665,3.64927],[30.5717,3.66673],[30.57924,3.73382],[30.55991,3.8113],[30.54817,3.83781],[30.52736,3.85862],[30.48641,3.83395],[30.4666,3.83429],[30.45821,3.85678],[30.43874,3.88128],[30.42733,3.86551],[30.4035,3.88027],[30.37362,3.90847],[30.3414,3.92659],[30.32798,3.9182],[30.30817,3.92626],[30.28199,3.94841],[30.2548,3.94707],[30.24104,3.93499],[30.20714,3.9417],[30.19271,3.98299],[30.19203,4.03249],[30.15948,4.05129],[30.16384,4.07512],[30.16216,4.11977],[30.14001,4.10869],[30.09536,4.12634],[30.05777,4.13373],[30.02656,4.18206],[30.02152,4.21227],[29.98728,4.21664],[29.97889,4.20657],[29.95674,4.22687],[29.94734,4.25406],[29.9705,4.29166],[29.96479,4.3024],[29.89968,4.34855],[29.84161,4.33613],[29.82818,4.3613],[29.79697,4.37272],[29.80166,4.40628],[29.79797,4.43884],[29.80871,4.47207],[29.80939,4.5251],[29.81931,4.55071],[29.80938,4.56455]]]},"properties":{"name":"Central Equatoria","id":"SS-32","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-32"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[31.79958,3.82704],[31.72429,4.00022],[31.76332,4.0295],[31.86415,4.03926],[31.9845,4.10431],[31.99751,4.24417],[32.04955,4.23767],[32.07232,4.27019],[32.06582,4.47836],[32.00727,4.5174],[32.15039,4.55318],[32.12436,4.59871],[32.17315,4.95325],[32.09834,5.21346],[32.10224,5.36788],[32.11442,5.85005],[32.51949,5.85147],[33.3116,5.84027],[33.73478,5.77959],[34.14426,5.72088],[35.01173,6.00819],[34.99853,5.98602],[34.99299,5.94728],[35.00205,5.93487],[34.99786,5.89859],[35.03962,5.84048],[35.1265,5.68785],[35.12705,5.6292],[35.30662,5.50337],[35.30919,5.46254],[35.29475,5.43228],[35.30029,5.39332],[35.29574,5.36405],[35.31057,5.33755],[35.3428,5.34032],[35.35483,5.35022],[35.40446,5.36376],[35.46535,5.40127],[35.47507,5.41261],[35.51212,5.42558],[35.54083,5.42488],[35.55842,5.41493],[35.58759,5.42071],[35.60218,5.40219],[35.62117,5.40543],[35.62557,5.38413],[35.65196,5.39363],[35.67859,5.38899],[35.70382,5.39664],[35.75244,5.37858],[35.77536,5.35218],[35.81889,5.34686],[35.86149,5.32231],[35.8601,5.29164],[35.82977,5.26362],[35.82885,5.23723],[35.84251,5.20968],[35.85987,5.19451],[35.8652,5.1702],[35.84019,5.12598],[35.81957,5.10835],[35.8246,4.78036],[35.9482,4.63346],[35.95099,4.62824],[35.62756,4.62899],[35.36416,4.6288],[35.00792,4.62862],[34.37747,4.63272],[34.30868,4.55908],[33.93863,4.17985],[33.66015,3.90002],[33.51213,3.7479],[33.17976,3.77614],[33.02708,3.88986],[32.89883,3.81343],[32.72286,3.76021],[32.41583,3.74473],[32.19642,3.59821],[32.21693,3.57659],[32.20332,3.50556],[32.0995,3.53178],[32.08135,3.56547],[32.06059,3.58691],[31.95458,3.58786],[31.94588,3.66988],[31.92104,3.71356],[31.88748,3.76252],[31.85776,3.79656],[31.82528,3.82196],[31.79958,3.82704]]]},"properties":{"name":"Eastern Equatoria","id":"SS-33","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-33"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[31.69732,5.87275],[31.02171,5.90953],[30.61188,6.25106],[30.52406,6.09168],[30.30394,6.08125],[30.19196,6.09325],[30.20721,6.15728],[30.15537,6.23657],[30.12183,6.35244],[29.9675,6.36367],[29.89922,6.3128],[29.91142,6.25486],[29.87178,6.26401],[29.85653,6.3128],[29.79249,6.30975],[29.78945,6.0841],[29.63088,5.97432],[29.56989,5.67548],[29.33592,5.6922],[29.01491,5.71513],[28.81975,5.81576],[28.65508,6.0597],[28.57275,6.07495],[28.52396,6.25181],[28.46297,6.29755],[28.49956,6.36769],[28.46602,6.48662],[28.49651,6.4979],[28.7923,6.38812],[28.89903,6.40642],[28.98746,6.53144],[29.00271,6.62902],[28.98136,6.76014],[28.86549,7.05593],[29.07894,7.42186],[29.32899,7.66581],[29.5333,7.7024],[29.69796,7.83261],[29.92667,7.79601],[29.97148,7.6916],[30.14418,7.29573],[30.2316,7.09466],[30.53654,7.06721],[30.61887,7.20139],[30.70621,7.21385],[31.19085,6.80727],[31.31271,6.60978],[31.47383,6.34864],[31.69732,5.87275]]]},"properties":{"name":"Lakes","id":"SS-24","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-24"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[27.89927,8.50032],[27.80736,8.49615],[27.72502,8.32843],[27.33776,8.19426],[26.92903,7.96694],[26.79497,7.89237],[26.27963,8.21256],[26.21425,8.60345],[26.13631,9.06943],[26.15021,9.20888],[26.21867,9.89578],[26.26665,9.74737],[26.31371,9.63662],[26.34793,9.59432],[26.39593,9.56581],[26.50953,9.52969],[26.59555,9.5083],[26.70392,9.48833],[26.74527,9.49594],[26.85316,9.53159],[26.91732,9.56248],[26.99242,9.59385],[27.02236,9.60193],[27.13738,9.62427],[27.23672,9.62569],[27.46723,9.62094],[27.75859,9.60335],[27.82845,9.60288],[27.90722,9.61016],[28.03606,9.34387],[27.86225,8.9566],[27.96592,8.87732],[27.99947,8.70045],[27.92323,8.65166],[27.95983,8.56323],[27.89927,8.50032]]]},"properties":{"name":"Northern Bahr el Ghazal","id":"SS-22","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-22"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[30.70621,7.21385],[30.61887,7.20139],[30.53654,7.06721],[30.2316,7.09466],[30.14418,7.29573],[29.97148,7.6916],[29.92667,7.79601],[29.69796,7.83261],[29.67596,8.14915],[29.65832,8.40284],[29.2741,8.69558],[29.09724,9.0371],[28.69472,9.02796],[28.7862,9.10511],[28.80755,9.30636],[28.76791,9.3521],[28.76791,9.41919],[28.8167,9.49847],[29.00037,9.67023],[29.01508,9.69684],[29.03745,9.70873],[29.05125,9.72967],[29.07838,9.75014],[29.17093,9.7478],[29.22971,9.74966],[29.53809,9.75013],[29.53809,10.00331],[29.53904,10.08183],[29.78317,10.20604],[29.93903,10.28664],[29.99677,10.2888],[30.1521,10.19196],[30.46583,10.00099],[30.53156,9.95945],[30.6351,9.87327],[30.82507,9.72378],[30.82733,9.71891],[30.84607,9.65334],[30.7615,9.61106],[30.75601,9.49297],[30.50584,9.51738],[30.35297,9.33848],[30.392,9.18561],[30.20985,8.96443],[30.26189,8.74325],[30.23587,8.70747],[30.32369,8.63266],[30.28466,8.55785],[30.37248,8.46352],[30.33215,8.23844],[30.33215,8.11158],[30.38419,8.11483],[30.54357,7.84486],[30.50779,7.65621],[30.67043,7.37323],[30.70621,7.21385]]]},"properties":{"name":"Unity","id":"SS-13","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-13"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[29.00037,9.67023],[28.8167,9.49847],[28.76791,9.41919],[28.76791,9.3521],[28.80755,9.30636],[28.7862,9.10511],[28.69472,9.02796],[29.09724,9.0371],[29.2741,8.69558],[29.65832,8.40284],[29.67596,8.14915],[29.69796,7.83261],[29.5333,7.7024],[29.32899,7.66581],[29.07894,7.42186],[28.86549,7.05593],[28.98136,6.76014],[29.00271,6.62902],[28.98746,6.53144],[28.89903,6.40642],[28.7923,6.38812],[28.49651,6.4979],[28.49414,6.76107],[28.49042,7.17486],[28.53616,7.43405],[28.29526,7.65971],[28.28916,7.7085],[28.31355,7.78168],[28.15194,8.02258],[27.86225,8.08357],[27.87749,8.27873],[27.91713,8.33057],[27.89927,8.50032],[27.95983,8.56323],[27.92323,8.65166],[27.99947,8.70045],[27.96592,8.87732],[27.86225,8.9566],[28.03606,9.34387],[27.90722,9.61016],[27.83298,9.7634],[27.83333,10.16667],[29,10.16667],[29.00037,9.67023]]]},"properties":{"name":"Warrap","id":"SS-21","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-21"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[27.89927,8.50032],[27.91713,8.33057],[27.87749,8.27873],[27.86225,8.08357],[28.15194,8.02258],[28.31355,7.78168],[28.28916,7.7085],[28.29526,7.65971],[28.53616,7.43405],[28.49042,7.17486],[28.49414,6.76107],[28.04826,6.73971],[27.93238,6.79765],[27.08362,6.71493],[26.41545,6.6498],[26.38608,6.66365],[26.38192,6.67879],[26.34107,6.69127],[26.33161,6.70224],[26.30324,6.70791],[26.26768,6.70375],[26.25292,6.71359],[26.25255,6.74385],[26.21926,6.74366],[26.20185,6.75085],[26.18634,6.77733],[26.17348,6.78111],[26.17159,6.81214],[26.14208,6.82197],[26.1069,6.81365],[26.08496,6.83937],[26.08193,6.88552],[26.05545,6.91957],[26.03805,6.92676],[26.05243,6.94605],[26.04373,6.95683],[26.04902,6.98558],[26.03729,7.00753],[25.98585,7.02039],[25.97979,7.04082],[25.93742,7.04157],[25.93478,7.05897],[25.90565,7.0637],[25.90792,7.08375],[25.87122,7.10494],[25.83718,7.10721],[25.81977,7.13104],[25.82053,7.15714],[25.80162,7.15298],[25.76227,7.16395],[25.75433,7.18287],[25.72293,7.17379],[25.69854,7.19666],[25.65929,7.19654],[25.63907,7.21065],[25.59958,7.2224],[25.59652,7.23674],[25.57278,7.24097],[25.56808,7.25813],[25.54787,7.25789],[25.53494,7.27858],[25.50414,7.26683],[25.48087,7.27129],[25.45713,7.28798],[25.4536,7.30702],[25.41554,7.33186],[25.39361,7.33186],[25.35403,7.35342],[25.33771,7.42536],[25.30963,7.43058],[25.26764,7.483],[25.23982,7.48032],[25.19923,7.50499],[25.18706,7.52751],[25.1968,7.54029],[25.17154,7.57571],[25.18554,7.60523],[25.20197,7.60614],[25.22449,7.63201],[25.24457,7.63444],[25.28565,7.65027],[25.29661,7.67644],[25.2787,7.71123],[25.28387,7.74927],[25.28498,7.8043],[25.25876,7.81057],[25.24694,7.8379],[25.21592,7.88222],[25.16348,7.9125],[25.15363,7.90506],[25.12573,7.91017],[25.10764,7.89924],[25.082,7.89848],[25.06654,7.91922],[25.02582,7.95447],[24.96173,7.98388],[24.95456,8.02008],[24.93232,8.0378],[24.90781,8.03704],[24.90065,8.07588],[24.88481,8.09209],[24.88934,8.11509],[24.85577,8.1798],[24.81699,8.18529],[24.79743,8.1949],[24.75075,8.18358],[24.74423,8.20177],[24.72193,8.21172],[24.69687,8.20451],[24.67422,8.21927],[24.62721,8.21618],[24.60833,8.22544],[24.56612,8.20108],[24.51533,8.21841],[24.4992,8.23591],[24.47998,8.27366],[24.41752,8.27915],[24.39281,8.27126],[24.38183,8.25238],[24.35541,8.24621],[24.33859,8.26817],[24.29878,8.26577],[24.27201,8.27401],[24.21711,8.31244],[24.19374,8.32491],[24.15464,8.37371],[24.15073,8.39965],[24.16708,8.42169],[24.16602,8.44586],[24.17348,8.48815],[24.20511,8.5045],[24.20653,8.52902],[24.23319,8.56456],[24.2584,8.57666],[24.2493,8.60282],[24.22428,8.6267],[24.22314,8.64073],[24.27243,8.70783],[24.30124,8.71617],[24.32158,8.73462],[24.36178,8.72691],[24.38776,8.73462],[24.40196,8.74761],[24.4048,8.79144],[24.42388,8.80808],[24.45676,8.79997],[24.45298,8.84489],[24.5449,8.8846],[24.57693,8.9208],[24.57518,8.95502],[24.54776,8.98576],[24.53832,9.04411],[24.54644,9.11201],[24.57277,9.1594],[24.55083,9.23366],[24.56355,9.25197],[24.56618,9.27962],[24.60874,9.31669],[24.61467,9.35925],[24.6366,9.39852],[24.65684,9.40293],[24.66613,9.41733],[24.66677,9.46123],[24.69253,9.4896],[24.70053,9.52417],[24.72006,9.54642],[24.72807,9.5708],[24.75613,9.6029],[24.75754,9.63842],[24.74503,9.65487],[24.75129,9.70251],[24.73736,9.72107],[24.79327,9.76467],[24.80699,9.76609],[24.83009,9.78564],[24.83757,9.80971],[24.88088,9.84459],[24.91135,9.85099],[24.92617,9.87198],[24.94364,9.87139],[24.95715,9.90178],[24.98621,9.93187],[24.9887,9.96098],[25.00632,9.96861],[25.01322,10.00252],[25.03685,10.02563],[25.03582,10.06086],[25.04801,10.07882],[25.07825,10.08483],[25.07179,10.09863],[25.07472,10.12726],[25.05447,10.15412],[25.03069,10.16061],[25.04889,10.17881],[25.05608,10.23341],[25.08368,10.30289],[25.09791,10.3161],[25.12566,10.32564],[25.18349,10.32263],[25.18936,10.34494],[25.22782,10.35639],[25.23678,10.34582],[25.27215,10.34039],[25.28932,10.35389],[25.32969,10.36461],[25.38797,10.38923],[25.47472,10.37425],[25.51163,10.39008],[25.56605,10.39648],[25.58198,10.40601],[25.61221,10.40194],[25.63465,10.40694],[25.65441,10.42496],[25.68034,10.40601],[25.71336,10.41985],[25.73266,10.41241],[25.76836,10.42659],[25.80126,10.4188],[25.8448,10.41682],[25.8448,10.4038],[25.90549,10.40183],[25.89851,10.38043],[25.93252,10.37523],[25.9311,10.18416],[25.98528,10.1642],[26.02711,10.13901],[26.04992,10.11714],[26.07606,10.06676],[26.11408,10.01163],[26.11599,9.99499],[26.21867,9.89578],[26.15021,9.20888],[26.13631,9.06943],[26.21425,8.60345],[26.27963,8.21256],[26.79497,7.89237],[26.92903,7.96694],[27.33776,8.19426],[27.72502,8.32843],[27.80736,8.49615],[27.89927,8.50032]]]},"properties":{"name":"Western Bahr el Ghazal","id":"SS-23","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-23"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[30.30394,6.08125],[30.65416,5.74365],[31.01846,5.57451],[30.99569,5.38586],[30.92413,5.38586],[30.93064,5.25575],[30.96317,5.20696],[30.93714,5.15166],[30.90462,5.15166],[30.89161,5.08336],[30.91112,5.03457],[30.86233,4.93048],[30.80379,4.93048],[30.70295,4.84917],[30.53707,4.80038],[30.5143,4.67027],[30.3972,4.60847],[30.29962,4.68328],[30.2053,4.7093],[30.07844,4.6475],[29.95484,4.65076],[29.80938,4.56455],[29.79779,4.55954],[29.76621,4.58759],[29.74336,4.57076],[29.71054,4.6158],[29.68375,4.62074],[29.65757,4.64442],[29.61623,4.66],[29.60066,4.65128],[29.55122,4.67059],[29.53344,4.65682],[29.51117,4.69056],[29.49322,4.69887],[29.48458,4.67214],[29.46497,4.66982],[29.47228,4.60816],[29.4776,4.5957],[29.44362,4.55716],[29.44813,4.52363],[29.40364,4.47479],[29.36527,4.47092],[29.35528,4.45125],[29.35399,4.42546],[29.34335,4.39515],[29.31949,4.38677],[29.28628,4.38709],[29.26919,4.39773],[29.25275,4.35291],[29.20987,4.34324],[29.19149,4.37033],[29.15764,4.39354],[29.13733,4.41546],[29.137,4.43448],[29.07867,4.43849],[29.0656,4.4581],[29.0429,4.46979],[29.03637,4.49283],[29.00198,4.49764],[28.99613,4.48595],[28.95796,4.48389],[28.93801,4.47529],[28.89289,4.48516],[28.88407,4.49368],[28.84481,4.48455],[28.83009,4.50646],[28.82949,4.52904],[28.8185,4.56702],[28.79383,4.57237],[28.7659,4.54741],[28.72667,4.54325],[28.72519,4.52452],[28.69814,4.51204],[28.7023,4.48768],[28.68923,4.47965],[28.67289,4.43255],[28.64079,4.41918],[28.60543,4.42185],[28.58849,4.39035],[28.57482,4.38292],[28.54455,4.38717],[28.51619,4.37938],[28.49284,4.32491],[28.45419,4.28688],[28.41854,4.28011],[28.3726,4.29215],[28.36783,4.3391],[28.35506,4.35435],[28.33456,4.3607],[28.28316,4.35967],[28.25602,4.35042],[28.20606,4.35186],[28.17625,4.38701],[28.13719,4.40367],[28.1499,4.42221],[28.1438,4.44153],[28.09085,4.43723],[28.0531,4.45698],[28.03704,4.49234],[28.04377,4.55548],[27.98173,4.55488],[27.94682,4.57529],[27.93972,4.55369],[27.87345,4.55502],[27.86161,4.56094],[27.83913,4.59526],[27.80126,4.60798],[27.80096,4.6281],[27.78262,4.65472],[27.78085,4.67721],[27.76576,4.68796],[27.78081,4.73673],[27.76042,4.78829],[27.74428,4.79332],[27.71559,4.78509],[27.69846,4.79966],[27.70344,4.834],[27.68236,4.85755],[27.67908,4.87608],[27.6448,4.89099],[27.56798,4.8947],[27.54853,4.90332],[27.52382,4.93972],[27.48823,4.96985],[27.47836,5.00392],[27.46418,5.00195],[27.44291,5.02036],[27.45828,5.05719],[27.45699,5.09688],[27.41677,5.12998],[27.40755,5.16902],[27.35512,5.20543],[27.33434,5.21371],[27.29115,5.28617],[27.28489,5.30668],[27.2595,5.3348],[27.25894,5.37529],[27.24312,5.40119],[27.2416,5.45108],[27.25158,5.4665],[27.26035,5.51005],[27.28031,5.5297],[27.28908,5.55435],[27.2788,5.56886],[27.29271,5.59215],[27.23132,5.61271],[27.24342,5.64258],[27.22134,5.67645],[27.22376,5.70095],[27.19866,5.72408],[27.19655,5.74071],[27.16722,5.74494],[27.16268,5.77095],[27.12216,5.79302],[27.10341,5.81026],[27.06833,5.79574],[27.01209,5.84452],[27.00181,5.86055],[26.96764,5.85601],[26.92984,5.87567],[26.93286,5.89562],[26.91533,5.90107],[26.85969,5.8923],[26.83549,5.89653],[26.82703,5.94794],[26.83459,5.97506],[26.8234,5.98927],[26.80223,5.98171],[26.75959,6.00832],[26.73571,6.00016],[26.69881,6.02012],[26.66071,6.00107],[26.64469,6.01558],[26.62654,6.01316],[26.60751,6.03321],[26.59249,6.02782],[26.55818,6.03746],[26.55563,6.05532],[26.53976,6.06666],[26.54004,6.10011],[26.51623,6.114],[26.47881,6.09954],[26.45499,6.07176],[26.45981,6.11684],[26.49468,6.13555],[26.51339,6.15341],[26.53324,6.18827],[26.53806,6.20925],[26.52615,6.23545],[26.50375,6.21645],[26.472,6.23006],[26.48389,6.26889],[26.47279,6.28758],[26.42724,6.30159],[26.396,6.30656],[26.34899,6.35897],[26.33147,6.36423],[26.29935,6.39693],[26.31687,6.43664],[26.29176,6.48642],[26.31512,6.49372],[26.32914,6.51679],[26.32826,6.53752],[26.35301,6.54094],[26.34906,6.57406],[26.37625,6.58938],[26.39107,6.61508],[26.40739,6.62151],[26.41545,6.6498],[27.08362,6.71493],[27.93238,6.79765],[28.04826,6.73971],[28.49414,6.76107],[28.49651,6.4979],[28.46602,6.48662],[28.49956,6.36769],[28.46297,6.29755],[28.52396,6.25181],[28.57275,6.07495],[28.65508,6.0597],[28.81975,5.81576],[29.01491,5.71513],[29.33592,5.6922],[29.56989,5.67548],[29.63088,5.97432],[29.78945,6.0841],[29.79249,6.30975],[29.85653,6.3128],[29.87178,6.26401],[29.91142,6.25486],[29.89922,6.3128],[29.9675,6.36367],[30.12183,6.35244],[30.15537,6.23657],[30.20721,6.15728],[30.19196,6.09325],[30.30394,6.08125]]]},"properties":{"name":"Western Equatoria","id":"SS-31","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-31"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[32.11442,5.85005],[31.69732,5.87275],[31.47383,6.34864],[31.31271,6.60978],[31.19085,6.80727],[30.70621,7.21385],[30.67043,7.37323],[30.50779,7.65621],[30.54357,7.84486],[30.38419,8.11483],[30.33215,8.11158],[30.33215,8.23844],[30.37248,8.46352],[30.28466,8.55785],[30.32369,8.63266],[30.23587,8.70747],[30.26189,8.74325],[30.20985,8.96443],[30.392,9.18561],[30.35297,9.33848],[30.50584,9.51738],[30.75601,9.49297],[30.93714,9.49721],[31.17847,9.42482],[31.45757,9.34109],[31.54539,9.38662],[31.6202,9.2923],[31.78284,9.27603],[31.81536,9.14593],[31.75681,9.08413],[31.85114,8.82391],[31.98125,8.71332],[32.09184,8.83367],[32.25772,8.928],[32.39108,9.09714],[32.42686,9.02233],[32.4106,8.928],[32.52119,8.90198],[32.64479,8.43685],[32.92777,8.27747],[32.83019,7.98798],[33.03565,7.98798],[33.03891,7.96778],[33.00245,7.94771],[33.00007,7.91547],[33.00706,7.85566],[33.04724,7.81331],[33.04909,7.79707],[33.07868,7.79707],[33.11701,7.7868],[33.18668,7.78906],[33.21279,7.77462],[33.23878,7.78079],[33.25194,7.76445],[33.30229,7.74333],[33.32325,7.70767],[33.34874,7.7273],[33.3659,7.72422],[33.38542,7.73912],[33.41615,7.73953],[33.43485,7.75566],[33.45187,7.73912],[33.46995,7.75093],[33.5019,7.72999],[33.52421,7.72847],[33.54133,7.70661],[33.58401,7.69212],[33.65197,7.69949],[33.69123,7.68485],[33.69017,7.66931],[33.72178,7.65824],[33.7368,7.62847],[33.75287,7.62663],[33.81478,7.59396],[33.84691,7.54589],[33.88195,7.51612],[33.91778,7.51744],[33.94913,7.51032],[33.94438,7.47239],[33.95887,7.44921],[33.98021,7.44249],[34.00787,7.40614],[34.00682,7.38269],[34.03527,7.36004],[34.02921,7.33001],[34.03954,7.271],[34.01374,7.25944],[34.04688,7.22296],[34.07513,7.21339],[34.11361,7.16913],[34.14141,7.14767],[34.16054,7.14722],[34.19035,7.13187],[34.19702,7.06381],[34.19546,7.03879],[34.22038,7.02789],[34.24351,7.00442],[34.25752,7.01287],[34.28555,6.99019],[34.29356,6.94726],[34.35272,6.9249],[34.43032,6.91681],[34.45878,6.9213],[34.48723,6.8906],[34.52504,6.8614],[34.53496,6.81545],[34.52953,6.75255],[34.5883,6.73032],[34.65144,6.73443],[34.6761,6.70193],[34.70487,6.69387],[34.71094,6.66632],[34.72682,6.64133],[34.76535,6.61202],[34.76722,6.59661],[34.79267,6.59755],[34.81532,6.60759],[34.84101,6.60338],[34.85479,6.61016],[34.89052,6.60023],[34.88351,6.58949],[34.92624,6.5596],[34.96524,6.51033],[35.01334,6.4704],[35.02128,6.43829],[35.01218,6.40957],[34.99163,6.38902],[34.9991,6.34967],[34.99163,6.32749],[34.96617,6.29526],[34.95847,6.24377],[34.97365,6.20758],[34.96337,6.12129],[34.97411,6.10121],[34.9998,6.08346],[35.0047,6.02543],[35.01173,6.00819],[34.14426,5.72088],[33.73478,5.77959],[33.3116,5.84027],[32.51949,5.85147],[32.11442,5.85005]]]},"properties":{"name":"Jonglei","id":"SS-12","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-12"},
{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[33.03565,7.98798],[32.83019,7.98798],[32.92777,8.27747],[32.64479,8.43685],[32.52119,8.90198],[32.4106,8.928],[32.42686,9.02233],[32.39108,9.09714],[32.25772,8.928],[32.09184,8.83367],[31.98125,8.71332],[31.85114,8.82391],[31.75681,9.08413],[31.81536,9.14593],[31.78284,9.27603],[31.6202,9.2923],[31.54539,9.38662],[31.45757,9.34109],[31.17847,9.42482],[30.93714,9.49721],[30.75601,9.49297],[30.7615,9.61106],[30.84607,9.65334],[30.82733,9.71891],[30.86133,9.73024],[30.89263,9.74859],[31.18081,9.74967],[31.25906,9.75615],[31.31465,9.77234],[31.35189,9.8031],[31.44795,9.92668],[31.51217,10.00386],[31.70213,10.21811],[31.7874,10.30175],[31.85756,10.40807],[31.97305,10.62718],[32.01028,10.66333],[32.14736,10.73727],[32.17488,10.76317],[32.37726,10.96555],[32.4404,11.02653],[32.47063,11.04272],[32.44634,11.08212],[32.40155,11.1744],[32.39345,11.21974],[32.39237,11.2764],[32.39399,11.57214],[32.39075,11.65093],[32.3845,11.69658],[32.36071,11.72371],[32.10658,11.945],[32.47492,11.9469],[32.74546,11.94758],[32.75505,11.97422],[32.75878,12.01258],[32.75078,12.04401],[32.75611,12.06692],[32.73906,12.13725],[32.74492,12.19266],[32.73054,12.23635],[33.02315,12.22673],[33.29774,12.21197],[33.25681,12.16581],[33.25015,12.12251],[33.22237,11.99279],[33.18383,11.78102],[33.17336,11.73866],[33.13529,11.62587],[33.14861,11.53974],[33.19715,11.24849],[33.23523,10.99722],[33.26568,10.82923],[33.21904,10.735],[33.48269,10.65791],[33.67209,10.43948],[33.72873,10.39046],[33.79267,10.3391],[33.90147,10.16155],[33.95097,10.16203],[33.96619,10.15394],[33.97095,10.09303],[33.9957,10.06209],[33.99237,10.04211],[33.99951,9.99975],[33.98856,9.96977],[33.99284,9.91076],[33.97714,9.8803],[33.96619,9.80606],[33.9443,9.77418],[33.91813,9.75657],[33.91004,9.71136],[33.89862,9.67995],[33.88767,9.6238],[33.88767,9.54623],[33.89957,9.49864],[34.10888,9.49885],[34.12593,9.32969],[34.14776,9.00566],[34.1483,8.85106],[34.14693,8.64534],[34.09089,8.64459],[34.05719,8.58948],[34.0434,8.53154],[34.02329,8.49017],[34.00184,8.48385],[33.96948,8.49381],[33.91049,8.48855],[33.89689,8.48357],[33.87621,8.43885],[33.84346,8.43617],[33.8291,8.42793],[33.81071,8.39729],[33.7929,8.39269],[33.7862,8.37239],[33.76972,8.36628],[33.75182,8.37874],[33.72983,8.3682],[33.70238,8.37687],[33.66403,8.4474],[33.62048,8.47114],[33.6017,8.45351],[33.57834,8.44732],[33.56103,8.47377],[33.52178,8.46799],[33.48905,8.47702],[33.48842,8.46264],[33.45924,8.45534],[33.44592,8.4439],[33.42283,8.45282],[33.39335,8.42648],[33.36921,8.44621],[33.33469,8.44789],[33.31085,8.46751],[33.28955,8.43992],[33.27486,8.46174],[33.25493,8.4481],[33.2176,8.40808],[33.19563,8.40893],[33.18745,8.38764],[33.19069,8.36039],[33.20977,8.33347],[33.19103,8.29783],[33.17297,8.30158],[33.17757,8.2532],[33.20568,8.23668],[33.19938,8.2159],[33.1711,8.20324],[33.19375,8.13152],[33.17365,8.13084],[33.16599,8.11687],[33.11972,8.11049],[33.12296,8.06551],[33.08497,8.07437],[33.07601,8.02097],[33.04467,8.01466],[33.03565,7.98798]]]},"properties":{"name":"Upper Nile","id":"SS-11","CNTRY":"South Sudan","TYPE":"State"},"id":"SS-11"}
]}
```

**Comment by martynasma** [comment]
Oh, you meant individual map for South Sudan :) Thanks man. We'll try to add that in the next release of the geodata package.

**Comment by strawberry-code** [comment]
If you need collaborators you can ask me. I am using amcharts but I noticed that many geojson are still missing at third level administrative divisions (counties). I am build my owns e.g. Italy counties (Province), Bosnia-E counties (Cantons) and so on.

**Comment by martynasma** [comment]
Thanks for the offer!

We're a bit reluctant on including a lot of local maps because that would quickly bloat up geodata package's size, which is already substantial.

We'll give it some thought tho. Maybe we could have some separate repo for contributed maps.

**Comment by martynasma** [comment]
South Sudan map (two of them actually) was included in the geodata 4.1.15 released yesterday.
